docs/components/gripper/_index.md

```diff
@@ -19,7 +19,8 @@ For configuration information, click on your gripper's model:
 
 Model | Description
 ----- | -----------
-[`softrobotics`](./softrobotics/) | The [*m*Grip soft gripper by Soft Robotics](https://www.softroboticsinc.com/products/mgrip-modular-gripping-solution-for-food-automation/)
+[`softrobotics`](./softrobotics/) | The [*m*Grip soft gripper by Soft Robotics](https://www.softroboticsinc.com/products/mgrip-modular-gripping-solution-for-food-automation/) |
+[`fake`](fake) | A model used for testing, with no physical hardware. |
```

Comment:
```suggestion
`fake` | A model used for testing, with no physical hardware. |
```
This would allow it to show up as a supported model before we add the fake page in